split-the-stream/README.md

```diff
@@ -0,0 +1,184 @@
+# Ideas about splitting the source-git work and upstream work
+
+Some ideas, possibilities, pros and cons of moving out the source-git related work.
+
+## What does the source-git workflow mean?
+
+Must have:
+
+- If user creates a merge-request on the source-git repository:
+  - Create a matching merge-request to the dist-git repository.
+  - Sync the CI results from the dist-git merge-request to the source-git merge-request.
+- If the dist-git is updated, update the source-git repository by opening a PR.
```

Comment:
I'd say this only applies to the specfile since synchronizing patches to the source-git repo would be immensely hard.